DXMainClient/PreStartup.cs

```diff
@@ -78,6 +78,14 @@ public static void Initialize(StartupParams parameters)
             try
             {
                 var translation = TranslationTable.LoadFromIniFile(ClientConfiguration.Instance.TranslationIniName);
+
+                if (translation is null)
+                {
+                    Logger.Log("Failed to load the translation file. File does not exist.");
+
+                    translation = new TranslationTable();
+                }
+
```

Comment:
Why? The original code is already handling this by catch{}.

Reply:
Getting a stack trace can be very performance intensive.
Try not to rely too much on `try catch` if you don't have to.

Reply:
Even though, the code to judge whether the file exists should be in `DXMainClient/PreStartup.cs` rather than `Localization/TranslationTable.cs`. An exception should be thrown in `LoadFromIniFile()` if file does not exist by design. The caller here is responsible for checking the file.

Reply:
<details>
<summary>Code and time consumption reference</summary>

![image](https://user-images.githubusercontent.com/42184238/175192658-c02cb422-e739-4126-8629-92b4d85fda39.png)
![image](https://user-images.githubusercontent.com/42184238/175192681-e29e5e81-ccd2-48f4-aff2-e2e1d38cece3.png)
![image](https://user-images.githubusercontent.com/42184238/175192606-17fdf4f1-4187-4a48-941b-b18df7d70af4.png)
</details>